.github/workflows/unit-test.yml

```diff
@@ -2,15 +2,38 @@ name: unit-test
 on:
   - pull_request
 jobs:
+  podman_tests:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+      - name: Uninstall docker
+        run: |
+          sudo apt-get update
+          sudo apt-get remove -y docker-ce
+      # podman should already be on the runner, but doesn't seem to be on the PATH
+      # Re-installing fixes this
+      - name: Install podman
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y podman
+      - run: bundle exec rspec spec/podman_spec.rb
   unix_tests:
-    runs-on: macos-latest
```

Comment:
Not exactly sure why this was ever set to macos, as that doesn't have a runtime installed. Switching over to ubuntu so we at least have one.